### PR TITLE
Contrib/CDS Bitsight release 1.1.2

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Changed grade filter argument from lowercase to uppercase for  bitsight-company-findings-get command.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Comment by kgal-pan** [comment]
Hi @crestdatasystems - Is there a reason the PR includes changes to 2 different Packs (BitSight, Chronicle)?

**Comment by crestdatasystems** [comment]
> Hi @crestdatasystems - Is there a reason the PR includes changes to 2 different Packs (BitSight, Chronicle)?

@kgal-pan It seems the branch was not synced with master now I've synced and it's showing BitSight changes only.

File: Packs/BitSight/ReleaseNotes/1_1_2.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### BitSight for Security Performance Management
+- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
+- In bitsight-company-findings-get command, grade argument is refactored to be in upper case.
```
Comment (JasBeilin) [inline Packs/BitSight/ReleaseNotes/1_1_2.md:5]:
```suggestion
- Note: the *grade* argument in **bitsight-company-findings-get** command is refactored to be in upper case.
```

**Review by JasBeilin — APPROVED** [review]

**Comment by JasBeilin** [comment]
Hi, This looks good. I had a small suggestion regarding the Release notes. Let me know if do not want to use the suggestion so we can merge.

**Comment by crestdatasystems** [comment]
> Hi, This looks good. I had a small suggestion regarding the Release notes. Let me know if do not want to use the suggestion so we can merge.

@JasBeilin Suggestion LGTM and updated the release notes.

**Pull request merged**